request-node/values.yaml

```diff
@@ -75,6 +81,8 @@ ipfs:
     service:
       type: ClusterIP
       port: 5001
+  livenessProbe: {}
```

Comment:
The default values are missing for the livenessProbe:
```
          initialDelaySeconds: 15
          timeoutSeconds: 5
          periodSeconds: 3
```

Reply:
it's on purpose actually, I think it's fine?